pkg/pool/operations/import.go

```diff
@@ -53,14 +56,18 @@ func (oc *OperationsConfig) Import(cspi *cstor.CStorPoolInstance) (bool, error)
 	klog.Infof("Importing pool %s %s", string(cspi.GetUID()), PoolName())
 	devID := pool.GetDevPathIfNotSlashDev(bdPath[0])
 	cacheFile := types.CStorPoolBasePath + types.CacheFileName
+	// existingPoolName denotes the pool name that may be present
+	// from previous version and needs to be imported with new name
+	existingPoolName := cspi.Annotations[types.OpenEBSCStorExistingPoolName]
 
 	// Import the pool using cachefile
 	// command will looks like: zpool import -c <cachefile_path> -o <cachefile_path> <pool_name>
 	cmdOut, err = zfs.NewPoolImport().
 		WithCachefile(cacheFile).
 		WithProperty("cachefile", cacheFile).
-		WithPool(PoolName()).
-		WithExecutor(oc.zcmdExecutor).
+		WithPool(existingPoolName).
+		WithNewPool(PoolName()).
```

Comment:
Good to put a comment of command if `existingPoolName` is set how it looks?

Reply:
Similarly in another place also

Reply:
Done